api/admin/controller.py

```diff
@@ -1030,6 +1046,228 @@ def edit_classifications(self, identifier_type, identifier):
 
         return Response("", 200)
 
+    def _validate_cover_image(self, image):
+        image_width, image_height = image.size
+        if image_width < 600 or image_height < 900:
+           return INVALID_COVER_IMAGE.detailed(_("Cover image must be at least 600px in width and 900px in height."))
+        return True
+
+    def _process_cover_image(self, work, image, title_position):
+        title = work.presentation_edition.title
+        author = work.presentation_edition.author
+        if author == Edition.UNKNOWN_AUTHOR:
+            author = ""
+
+        if title_position in ["top", "center", "bottom"]:
```

Comment:
This should also be constants, especially because it's not exhaustive -- 'none' is also possible but I didn't learn this until later.